### PR TITLE
Work experience, Projects & Media Queries

### PR DESCRIPTION
**Pull request by anushadatta**
Added work experience and projects, and a parallax scroll image between those two sections. Also added relevant media queries to make the webpage responsive.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -116,6 +116,137 @@ <h4 class="title">Bachelor of Eng. : Computer Science (2018 - Present)</h4>
...
+                        <li>Invited to prestigious President’s Undergraduate Research Program offered to top 10% of cohort.</li>
+                        <li>Developed a new deep learning model for stock price prediction, using Sentiment Analysis & Natural Language Processing with high accuracy of 85%.</li>
+                    </ul>
+                </div>
+
+                <!-- Work experience 2 -->
+                <div>
+                    <h3>VANTAGE WEBTECH</h3>
+                    <br>
+                    <h4>Android application Developer</h4>
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/index.html:156]: ditto, I think heading tags already start next elemnt at next line. Or consider using styling element instead.

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -148,6 +148,121 @@ hr {
...
+    color: grey;
+    text-decoration-style: solid;
+}
+
+#projects a:hover {
+    color: darkgray;
+}
+
+/* PROJECT CARDS */
+
```
Comment (rashikaranpuria) [inline portfolio/src/main/webapp/style.css:206]: nit, Unnecessary newline after comment. Because of new line styles below seem unrelated to comment :D

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/index.html:156]: Oops, okay I'll add it to a styling element.

File: portfolio/src/main/webapp/style.css (hunk 2 above)
Comment (anushadatta) [reply portfolio/src/main/webapp/style.css:206]: You're right I didn't realize that, I'll get rid of this extra newline right away!
